docs/src/content/docs/fr/guides/components.mdx

```diff
@@ -423,6 +422,36 @@ import { Steps } from '@astrojs/starlight/components';
 
 </Steps>
 
+### Badges
+
+import { Badge } from '@astrojs/starlight/components';
+
+Utilisez le composant `<Badge>` pour afficher de petits éléments d'information, tels que des statuts ou des étiquettes.
+
+Passez le contenu que vous souhaitez afficher à l'attribut `text` du composant `<Badge>`.
+
+Par défaut, le badge utilise la couleur d'accentuation du thème de votre site. Pour utiliser une couleur de badge intégrée, définissez l'attribut `variant` à l'une des valeurs suivantes : `note` (bleu), `tip` (violet), `danger` (rouge), `caution` (orange), ou `success` (vert).
+
+L'attribut `size` (par défaut : `small`) contrôle la taille du texte du badge. Les attributs `medium` et `large` sont également disponibles pour afficher un badge plus grand.
+
+Pour une personnalisation plus poussée, utilisez d'autres attributs `<span>` tels que `class` ou `style` avec du CSS personnalisé.
```

Comment:
```suggestion
Pour une personnalisation plus poussée, utilisez d'autres attributs de  l'élément `<span>` tels que `class` ou `style` avec du CSS personnalisé.
```

Not sure this reads as well in French as in English so I think adding a bit more context makes sense here.